src/main/java/cubicchunks/world/cube/Cube.java

```diff
@@ -56,14 +56,13 @@
 import net.minecraftforge.event.entity.EntityEvent;
 import org.apache.logging.log4j.Logger;
 
+import javax.annotation.Nullable;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ConcurrentLinkedQueue;
 
-import javax.annotation.Nullable;
```

Comment:
Same reason, I figure. I refactored stuff, and IDEA uses that opportunity to reformat imports. Should be according to the project's format settings either way.